subscriptions/configuration/subscription_templates.rst

```diff
@@ -0,0 +1,66 @@
+==========================
+Use subscription templates
+==========================
+
+The subscription business model is becoming more and more popular. Are you wondering why? Because a
+subscription model simplifies the business process and makes it very easy for both customers and the
+business. It ensures that the business retains more customers on a consistent basis and that clients
+can be reached out to and engaged with regularly. A business model like this makes it easy to
+anticipate demand and supply. Finally, this model offers much higher payment security for your
+business.
+
+.. raw:: html
+
+   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>Simplicity.
+   Higher customer retention. Opportunities for marketing. Business consistency. Better cash flow
+   management.</b> </div>
+
+Odoo subscription templates will help you save a lot of time and money. Indeed, these templates will
+help you generate recurring invoices and manage renewals at a fast pace. With Odoo you have the
+possibility to create, edit and manage your own subscription templates.
+
+Configuration
+=============
+
+Go to :menuselection:`Subscriptions → Configuration → Subscription templates`. By default, Odoo
+suggests you two types of subscription (MON - Monthly subscription *vs* YEA - Yearly subscription).
+Of course, you can create your own ones.
+
+.. image:: media/subscription_templates_1.png
+  :align: center
+  :alt: How to use subscription templates on Odoo Subscriptions?
```

Comment:
Reminder about the alt tags: we actually use them to describe the image, with a few words, as if we would for a blind person (: